conans/client/cmd/uploader.py

```diff
@@ -501,31 +509,42 @@ def compress_files(files, symlinks, name, dest_dir, output=None):
         # tgz_contents = BytesIO()
         tgz = gzopen_without_timestamps(name, mode="w", fileobj=tgz_handle)
 
-        for filename, dest in sorted(symlinks.items()):
-            info = tarfile.TarInfo(name=filename)
+        for relpath, dest in sorted(gathered_files.symlinks.items()):
+            if dest.startswith("."):
```

Comment:
why ignore files with dot here? what if I package something like `.bashrc`, `.vimrc`, etc?

Reply:
Genius! Not only a problem here but in other places where we are doing the same. Thanks!